app/views/layouts/components_previews.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Here we can see the new layout for previews. As we see this is so simple layout, with almost nothing more than load CSS and HTML.

Something that I added is `<%= params[:lookbook][:display][:body_classes] %>` that will give us the ability to edit the body classes using tailwind from the preview using `@display`.

So for example if in a component preview we want a different background just because make sense in that context, we can do something like:

```ruby
@display body_classes "bg-gray-500"
```

Reply:
Great call. I experimented with something like this when I fist played with Lookbook but I like the simplicity of your approach better. We might want to standardize it a bit more with "wide", "narrow", or "dark", etc. But this works for now.

Please use `<%= tag.body class: params[:lookbook][:display][:body_classes] %>` to remove the interpolated string.